docs/Support_Policy.md

```diff
@@ -24,10 +24,10 @@ SmokeAWS has two mechanism for removing support for Swift Toolchain versions-
 Once support has been removed, testing via continuous integration may be removed and no guarantees will be given for compiling the package using this version of the toolchain.
 
 Following these rules, the support level for different Swift Toolchain versions are-
-1. **Swift 5.3 and earlier**: No support.
-2. **Swift 5.4**: Supported for SmokeAWS 2.x. Support to be removed after 14th of September, 2022.
-3. **Swift 5.5**: Supported for SmokeAWS 2.x.
-4. **Swift 5.6**: Supported for SmokeAWS 2.x.
+1. **Swift 5.4 and earlier**: No support.
+2. **Swift 5.5**: Supported for SmokeAWS 2.x. Support to be removed after 12th of March, 2023.
+3. **Swift 5.6**: Supported for SmokeAWS 2.x.
```

Comment:
Maybe a comment to indicate that not all platform are supported.

Reply:
Added link to compiler crash issue for Swift 5.6.